### PR TITLE
Jsdatafix

### PR DESCRIPTION
**Pull request by ppatrzyk**
RE: https://github.com/iheanyi/bandcamp-dl/issues/166

Apparently Bandcamp changed data structure on their pages. I played around a little bit with code that collects JSON data strings from page html and managed to get a version that works as expected.

### CODE REVIEW

**Review by anio — APPROVED** [review]

File: bandcamp_dl/bandcampjson.py [hunk 1]
```diff
@@ -7,48 +7,41 @@
...
-
-    def extract_data(self, js: str):
-        """Extract values from JS dictionary
-
-        :param js: Raw JS
-        """
-        self.js_data = self.regex.search(js).group().replace('" + "', '') + "}"
-
-    def js_to_json(self):
+        logging.debug("Grabbing embedded scripts..")
```
Comment (Evolution0) [inline bandcamp_dl/bandcampjson.py]: Logging text is pre-spaced for readability if I remember correctly.

File: bandcamp_dl/bandcampjson.py [hunk 2]
```diff
@@ -7,48 +7,41 @@
...
-                self.js_to_json()
+    def generate(self):
+        """Grabbing needed data from the page"""
+        self.get_pagedata()
+        self.get_js()
         return self.json_data
 
     def get_pagedata(self):
-        """Grab bandcamp pagedata JSON"""
+        logging.debug("Grab pagedata JSON..")
```
Comment (Evolution0) [inline bandcamp_dl/bandcampjson.py]: Logging spacing again.

**Comment by Evolution0** [comment]
Just fix the debug logging to remain consistent with the rest of the code and I'll merge it

**Comment by ppatrzyk** [comment]
@Evolution0 sure, I changed that

**Pull request merged**